README.md

```diff
@@ -2,42 +2,40 @@
 
 [![Build Status](https://travis-ci.org/operator-framework/operator-sdk.svg?branch=master)](https://travis-ci.org/operator-framework/operator-sdk)
 
-### Project Status: pre-alpha
+### Project Status: alpha
 
-The project is currently pre-alpha and it is expected that breaking changes to the API will be made in the upcoming releases.
+The project is currently alpha. The core APIs provided by the controller-runtime will most likely be unchanged. New features and APIs may still be added in the future.
```

Comment:
Probably shouldn't mention controller-runtime before defining what it is below, or link it up here too.

Reply:
There is talk about adding api breaking changes to the controller-runtime [see](https://github.com/kubernetes-sigs/controller-runtime/pull/154#issuecomment-424381319) just an FYI